modules/universal/reaper/provision/docker-compose.yml

```
@@ -0,0 +1,31 @@
+version: '3.7'
+services:
+  cqlsh:
+    image: scalar-cassandra-reaper:latest
+    build:
+      context: ./
+    command:
+     - "cqlsh"
+     - "--cqlversion=3.4.4"
+     - "--username=${REAPER_CASS_AUTH_USERNAME}"
+     - "--password=${REAPER_CASS_AUTH_PASSWORD}"
+     - "--execute"
+     - "CREATE KEYSPACE reaper_db WITH replication = {'class': 'SimpleStrategy', 'replication_factor': ${CASSANDRA_REPLICATION_FACTOR} };"
```

```suggestion
     - "CREATE KEYSPACE IF NOT EXISTS reaper_db WITH replication = {'class': 'SimpleStrategy', 'replication_factor': ${CASSANDRA_REPLICATION_FACTOR} };"
```

I did a quick check and `CREATE KEYSPACE` returns 2 if there is the keyspace already. When specified with `IF NOT EXISTS`, it returns 0 even if there is the keyspace.

I am wondering if it will keep restarting the container without `IF NOT EXISTS` when C* password is changed.

---

@feeblefakie Right. I noticed that the cqlsh container that runs `CREATE KEYSPACE` keeps restarting when it is run second time, so I was going to report a separate issue. Simply adding `IF NOT EXISTS` should solve it, should I apply the fix here?

---

It can be separated but I think it's fine to apply this here.